src/features/home/courses/courses.tsx

```diff
@@ -23,18 +23,32 @@ const courses: CourseProps[] = [
     icon: <AwsLogo />,
   },
   {
-    title: 'JS / Front-end. Pre-school',
+    title: 'AWS Cloud Developer',
```

Comment:
I suppose, we need one data-source for all courses components (e.g. `courses.data.ts`), and we have to remove all data from components and tests.

Reply:
Yes, these changes are planned as part of the 61 issue: https://github.com/rolling-scopes/site/issues/61

Reply:
I suppose, it doesn't quite fit the task #61 . You can change in this PR to foment PR 61 task instead